tests/unit/algo/classification/test_torchvision_model.py

```diff
@@ -45,7 +45,7 @@ class TestOTXTVModel:
     def test_create_model(self, fxt_tv_model):
         assert isinstance(fxt_tv_model.model, TVClassificationModel)
 
-        semi_sl_model = OTXTVModel(backbone="mobilenet_v3_small", label_info=10, train_type="semi_supervised")
+        semi_sl_model = OTXTVModel(backbone="mobilenet_v3_small", label_info=10, train_type="SEMI_SUPERVISED")
```

Comment:
Just a question, why did you use string instead of enum?

Reply:
Here we can use both enum and string at the same time. My guess is that the average user will use str more.